metis-debias/metis-debias-detect-service/src/main/java/eu/europeana/metis/debias/detect/model/request/DetectionParameter.java

```diff
@@ -0,0 +1,51 @@
+package eu.europeana.metis.debias.detect.model.request;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * The type Detection parameter.
+ */
+public class DetectionParameter {
+
+  private String language;
+  private List<String> values;
+
+  /**
+   * Gets language.
+   *
+   * @return the language
+   */
+  public String getLanguage() {
+    return language;
+  }
+
+  /**
+   * Sets language.
+   *
+   * @param language the language
+   */
+  public void setLanguage(String language) {
```

Comment:
Would it not be better if this method accepted an enum? Or (in case this is considered to be inconsistent) perhaps mention in the javadoc that the 2-letter ISO values are expected here. Same with the getter.